src/components/OpenSource/Talks.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+
+import BlueBackground from '../Common/BlueBackground'
+import CompactVideoLink from '../Common/CompactVideoLink'
+import StyledLink from '../Common/StyledLink'
+import { Grid, Row, Col } from '../grid'
+import { SectionTitle } from '../Typography'
+
+const StyledImage = styled.img`
+  padding: ${remcalc(83)} 0 ${remcalc(15)};
```

Comment:
Where are these values coming from?

You need `72px` above and below the whole component (i.e. applied to `Grid`) - please use the `space` array in our theme & don't forget that other breakpoints will have different values.

The only other padding you might want to apply to the image is in order to achieve the `24px` between the image and the title below it:

![image](https://user-images.githubusercontent.com/15656538/58719790-6ef8ae80-83c8-11e9-8b0d-1db1ea623f66.png)

Reply:
Yeah, pretty much. I wished I knew . that before; would have saved some time to all of us.

Reply:
Done. Thanks Jen!